app/svg_test.go

```diff
@@ -64,6 +64,7 @@ func TestParseInvalidSVGData(t *testing.T) {
 	invalidSVGs := []io.Reader{
 		generateSVGData(width, height, false, false, false), // missing viewBox, width & height
 		generateSVGData(width, 0, false, true, false),       // missing viewBox, malformed width & height
+		generateSVGData(0, height, false, true, false),      // missing viewBox, malformed width & height
```

Comment:
This seems unrelated?

Reply:
In every case `width` is same that's why the error `always receives ...`.

Reply:
Let's add an ignore statement for this now, and fix this in a separate PR.